src/main/java/net/honux/chess/Board.java

```diff
@@ -3,20 +3,82 @@
 import net.honux.chess.pieces.Pawn;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class Board {
 
-    ArrayList<Pawn> pawns = new ArrayList<Pawn>();
+    List<Pawn> whitePawns = new ArrayList<Pawn>();
+    List<Pawn> blackPawns = new ArrayList<Pawn>();
 
-    public void add(Pawn pawn) {
-        pawns.add(pawn);
+
+    public void whitePawnsAdd(Pawn pawn){
+        whitePawns.add(pawn);
+    }
+
+    public void blackPawnsAdd(Pawn pawn){
+        blackPawns.add(pawn);
     }
 
     public int size() {
-        return pawns.size();
+        return whitePawns.size()+blackPawns.size();
+    }
+
+    public Pawn findWhitePawn(int index) {return whitePawns.get(index);}
+
+    public Pawn findBlackPawn(int index) {
+        return blackPawns.get(index);
     }
 
-    public Pawn findPawn(int index) {
-        return pawns.get(index);
+    public void initialize() {
+        int boardSize = 8;
+
+        for (int i = 1; i <= boardSize; i++) {
+            whitePawnsAdd(new Pawn(Pawn.WHITE_COLOR,Pawn.WHITE_REPRESENTATION));
+            blackPawnsAdd(new Pawn(Pawn.BLACK_COLOR,Pawn.BLACK_REPRESENTATION));
+        }
```

Comment:
```suggestion
        for (int i = 0; i < boardSize; i++) {
            whitePawnsAdd(new Pawn(Pawn.WHITE_COLOR,Pawn.WHITE_REPRESENTATION));
            blackPawnsAdd(new Pawn(Pawn.BLACK_COLOR,Pawn.BLACK_REPRESENTATION));
        }
```
저는 보통 index를 0부터 시작하는 방법을 선호합니다. 이 방법에 익숙해지시는게 좋을 것 같아요!

Reply:
넵넵 그러네요 ㅎㅎ index는 0 부터 시작하도록 변경하겠습니다 :)